servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1319,7 +1319,7 @@ void RenderForwardClustered::_render_scene(RenderDataRD *p_render_data, const Co
 
 	_fill_render_list(RENDER_LIST_OPAQUE, p_render_data, PASS_MODE_COLOR, using_sdfgi, using_sdfgi || using_voxelgi);
 	render_list[RENDER_LIST_OPAQUE].sort_by_key();
-	render_list[RENDER_LIST_ALPHA].sort_by_reverse_depth_and_priority();
+	render_list[RENDER_LIST_ALPHA].sort_by_reverse_depth_and_priority(RenderList::TransparencySortMode::Z_AXIS);
```

Comment:
PR is draft because this is hard-coded, as I don't know how to pass the option through to this part (I suppose this should be a project setting or a viewport or camera option).